src/sentry/seer/explorer/models.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
```

Comment:
nit: not needed?

Reply:
it's needed for the `children: list[ExecutionTreeNode]` field inside the `ExecutionTreeNode` model